config/helm/aws-node-termination-handler/Chart.yaml

```diff
@@ -1,21 +1,25 @@
-apiVersion: v1
+apiVersion: v2
 name: aws-node-termination-handler
-description: A Helm chart for the AWS Node Termination Handler
-version: 0.16.0
+description: A Helm chart for the AWS Node Termination Handler.
+type: application
+version: 0.17.0
```

Comment:
can you leave this at 0.16.0? During our release process, it will get updated. We'll do a minor version release next and it will get incremented to `0.17.0` w/ the NTH `v1.15.0` release.

Reply:
Done